plugins/Hosting/js/reducers/hosting.js

```diff
@@ -0,0 +1,96 @@
+import { Map, List } from 'immutable'
+import * as constants from '../constants/constants.js'
+
+const initialState = Map({
+	numContracts: 0,
+	storage: 0,
+	usersettings: List([
+		Map({ name: 'Max Duration (Weeks)', value: 0, min: 12 }),
+		Map({ name: 'Collateral per TB per Month (SC)', value: 0 }),
+		Map({ name: 'Price per TB per Month (SC)', value: 0 }),
+		Map({ name: 'Bandwidth Price (SC/TB)', value: 0 }),
```

Comment:
What if we had a `settings` reducer that had `maxduration`, `collateral`, `price`, `bandwidthprice`?  Seems much clearer and less error prone to me